static/local/blackwomenssuffrage/timeline.md

```diff
@@ -1,327 +1,328 @@
+A second black sorority, Delta Sigma Theta, is founded at Howard University in order to take more overtly political actions. Mary Church Terrell served as a mentor for the Deltas and wrote their vow. 
 
-Ida Wells Barnett founds the [Alpha Suffrage Club](/search?q=Alpha+Suffrage+Club).
-
-Black women found the [Women's Cooperative Civic League](/search?q=Women's+Cooperative+Civic+League) to address problems of housing, health, sanitation and education in Baltimore due to rapid growth. Leaders divide the cities into wards, with heads that report on problems to the city government, helping to ensure the needs of Black communities were being met.
+Black women found the Women's Cooperative Civic League to address problems of housing, health, sanitation and education in Baltimore due to rapid growth. Leaders divide the city into wards, with heads that report on problems to the city government, helping to ensure the needs of Black communities are being met. 
 
 **1914**: NACW holds its ninth biennial meeting. At this point, the NACW represents 50,000 Black women in 28 state federations and more than 1,000 local groups.
 
-**1914-15**: Ida B. Wells Barnett and friends "mobilize" Alpha Suffrage Club in the wake of "expanded woman suffrage rights." The group works to register votes to try to elect black aldermen. In 1914, [Oscar DePriest](/search?q=Oscar+DePriest) solicits their help in running for open alderman seat. (Terborg-Penn, p. 139)
-
-**1915**: 40,000 women march in a suffrage parade in New York City.
+**1914&#8211;15**: Ida B. Wells Barnett and members of the Alpha Suffrage Club work to register votes to try to elect black aldermen in Chicago. In 1914, Oscar DePriest solicits their help in running for an open alderman seat (Terborg-Penn, 139). 
 
-NAACP leader Mary Talbert writes an article for the August 1915 issue of the NAACP publication, The Crisis entitled "Women and Colored Women" saying, "...this struggle becomes two-fold, first, because we are women and second, because we are colored women…" In an essay in that same issue, Nannie Helen Burroughs responded to a white woman asking what Black women would do with the ballot: "What can she do without it?"
+**1915**: 40,000 women, including Black women and immigrant factory workers, march in a suffrage parade in New York City. 
 
-[Josephine Bruce](/search?q=Josephine+Bruce), wife of Mississippi's first Black senator Blanche K. Bruce, pens editorials for NACW in which she advocates for women's suffrage.
+NAACP leader [Mary Talbert](/search?q=Mary+Talbert) writes an article for the August 1915 issue of the NAACP publication, The Crisis entitled "Women and Colored Women" saying, "...this struggle becomes two-fold, first, because we are women and second, because we are colored women&#8230;" In an essay in that same issue, [Nannie Helen Burroughs](/search?q=Nannie+Helen+Burroughs) responded to a white woman asking what Black women would do with the ballot: "What can she do without it?" 
```

Comment:
Change link to `/search?q=Nannie+Burroughs` (for consistency with the other time her name is linked on this page).